scripts/screeps-runtime-monitor.py

```
@@ -1694,6 +1699,71 @@ def should_bypass_alert_debounce(reason: dict[str, Any]) -> bool:
     return alert_reason_kind(reason) in DEBOUNCE_BYPASS_REASON_KINDS
 
 
+def expected_rampart_decay_delta(previous_room_state: dict[str, Any], current_tick_value: Any) -> int:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `current_tick_value`

(ANN401)